www/assets/js/lib/util.ts

```diff
@@ -33,3 +35,20 @@ export const sentryCaptureException = (excetpion: any) => {
 export const sentryCaptureMessage = (message: string) => {
   Sentry.captureException(message)
 }
+
+/**
+ * Converts course list array into learning resources array while maintaing order
+ */
+export const getLearningResourcesFromCourseList = (
+  courseList: CourseJSONMap[]
+): LearningResource[] => {
+  const numberOfCourses = courseList.length || 0
+  const learningResources: LearningResource[] = []
+  for (let i = 0; i < numberOfCourses; i++) {
```

Comment:
I think we can still use `map` here, no?  Was `map` not maintaining the order? If you need the index from `map`, it will be the second argument passed to the function call. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Reply:
Yeah, I was just being a bit extra-careful :P haha
but `map` is good, thanks for pointing this out. 
I have updated the PR